packages/main/src/themes/SliderBase.css

```diff
@@ -21,12 +21,16 @@
 .ui5-slider-root {
 	box-sizing: border-box;
 	height: 3.3125rem;
-	padding: 1rem 0;
+	padding: 1rem;
 	outline: none;
 	touch-action: none;
 	-ms-touch-action: pan-y;
 }
 
+:host-context([ui5-color-picker]) .ui5-slider-root {
```

Comment:
Unfortunately :host-context is not supported on FF and Safari and we can't use it.